packages/sdks-tests/src/e2e-tests/custom-components.spec.ts

```diff
@@ -33,6 +33,14 @@ test.describe('Custom components', () => {
     await customComponentMsgPromise;
   });
 
+  test('children placement is correct', async ({ page, sdk }) => {
+    test.skip(!['angular'].includes(sdk));
```

Comment:
shouldn't this test work on all SDKs?

Reply:
oh nevermind. its testing a custom component

Reply:
the `hello` custom component is there only in Angular SDK as of now, but yes, the test should pass in all SDKs